recipes-bsp/uefi/edk2_git.bb

```diff
@@ -60,6 +60,13 @@ do_compile() {
 
 do_deploy() {
     # Placeholder to be implemented in machine specific recipe
+
+    if [ -e ${EDK2_DIR}/atf/build/${UEFIMACHINE}/release/bl1.bin ] ; then 
```

Comment:
why do we need bl1.bin? we don't need to deploy it, it's only used to build fip.bin.